### PR TITLE
RHEL-9.0: install and enable TuneD by default on all EC2 images

### LINKED ISSUE #1972: [RHEL-9] Install and enable tuned service on all RHEL EC2 images
Based on discussion with Virt devel and QE, the request is to:
**Install and enable tuned on RHEL-9 EC2 and EC2-HA images**

 The RHEL EC2-SAP image already has Tuned installed and enabled. In addition it activates an appropriate SAP profile.

Related to RHELPLAN-102615

### PR DESCRIPTION
**Pull request by thozza**
Add TuneD package to the base package set for all EC2 image types,
including the `ami` image type. In addition to installing the package,
also enable the service by default. TuneD will by default auto-detect
the environment in which the image is running and set the most
appropriate TuneD profile, with exception of the `ec2-sap` image, which
explicitly sets a specific TuneD profile.

This change affects the `ami`, `ec2`, and `ec2-ha` image types on all
supported architectures.

Regenerate affected image test cases.

Related to RHELPLAN-102615
Fix #1972

Signed-off-by: Tomas Hozza <thozza@redhat.com>

This pull request includes:

- [x] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Review by teg — APPROVED** [review]
Love the increased uniformity! Should we enable this on any other image types too?

**Review by achilleas-k — APPROVED** [review]

**Comment by thozza** [comment]
Blocked on #2049...

**Comment by thozza** [comment]
Rebased after #2049 got merged. There were no other changes

**Review by ondrejbudai — APPROVED** [review]
Great!

image

**Comment by thozza** [comment]
> Love the increased uniformity! Should we enable this on any other image types too?

Personally I think that installing and enabling TuneD by default makes sense, since it will tune the system for the environment in which it is being used.

I checked the situation and the outcome was interesting... TuneD was in the `@Core` group on RHEL-8, but got removed from it on RHEL-9. So basically the majority of images on RHEL-8 (`ami`, `ec2`, `ec2-ha`, `ec2-sap`, `edge-commit-rt`, `qcow2`, `openstack`, `vhd`, `vmdk`) have it installed, while on RHEL-9 only AWS images and the `edge-commit-rt` images have it. I think that it would make sense to keep it installed on the RHEL Guest image. I'll ping appropriate people...

**Pull request merged**
